__test__/pet.test.js

```diff
@@ -0,0 +1,156 @@
+const Pet = require("../src/pet");
```

Comment:
A thought about the constants...
Rather than storing them in the pet.js file and having to replicate them here, it would be even more useful if they were to be stored in their own file and imported in, as this would allow them to be used in the test files as well. This would mean you dont have to declare them twice and, and would make the code more maintainable in the long run as if these values were to change at some point in the future it would only mean changing them in one place:

EG constants.js:
```
const MINIMUM_HUNGER = 0;
const MAXIMUM_FITNESS = 10;
const INITIAL_AGE = 0;
const INITIAL_HUNGER = 0;
const INITIAL_FITNESS = 10;
const GROWUP_AGE = 1;
const GROWUP_HUNGER = 5;
const GROWUP_FITNESS = 3;
const WALK_FITNESS = 4;
const FEED_HUNGER = 3;

exports.MINIMUM_HUNGER = MINIMUM_HUNGER;
exports.MAXIMUM_FITNESS = MAXIMUM_FITNESS;
exports.INITIAL_AGE = INITIAL_AGE;
exports.INITIAL_HUNGER = INITIAL_HUNGER;
exports.INITIAL_FITNESS = INITIAL_FITNESS;
exports.GROWUP_AGE = GROWUP_AGE;
exports.GROWUP_HUNGER = GROWUP_HUNGER;
exports.GROWUP_FITNESS = GROWUP_FITNESS;
exports.WALK_FITNESS = WALK_FITNESS;
exports.FEED_HUNGER = FEED_HUNGER;
```

These could then be imported like:
```
const {
  MINIMUM_HUNGER,
  MAXIMUM_FITNESS,
  INITIAL_AGE,
  INITIAL_HUNGER,
  INITIAL_FITNESS,
  GROWUP_AGE,
  GROWUP_HUNGER,
  GROWUP_FITNESS,
  WALK_FITNESS,
  FEED_HUNGER,
} = require("./constants");
```
and referenced in this file as well as pet.js